src/components/MenuBar/MenuBar.tsx

```diff
@@ -124,7 +124,7 @@ export default function MenuBar() {
               type="submit"
               color="primary"
               variant="contained"
-              disabled={isConnecting || !name || !roomName || isFetching}
+              disabled={isAcquiringLocalTracks || isConnecting || !name || !roomName || isFetching}
```

Comment:
Instead of adding another flag, is it possible to check for the existence of the local track instead?

Reply:
If the user disables their video, then there would be no video track, but we would still want the user to be able to join the room.  We also want the user to be able to join the room if they deny media permissions.

So this flag indicates whether or not the permissions dialog is still open, not whether or not there are local tracks.